o8g/Sets/Devil Reef/set.xml

```diff
@@ -239,11 +239,12 @@
       <property name="Encounter Set" value="Devil’s Reef" />
       <property name="Type" value="Location" />
       <property name="Traits" value="Ocean." />
+      <property name="Text" value="Churning Waters enters play fully flooded (even while it is unrevealed)" />
       <alternate name="Churning Waters" type="B" size="EncounterCard">
         <property name="Encounter Set" value="Devil’s Reef" />
         <property name="Type" value="Location" />
         <property name="Traits" value="Ocean." />
-        <property name="Text" value="Churning Waters enters play fully flooded.η If you are in Fishing Vessel: Resign. Each investigator in Fishing Vessel resigns. Remove Fishing Vessel from the game. Each resigning investigator places each of their purple, black, or white keys on the current act. All other investigators are left to fend for themselves." />
```

Comment:
There is a disparity between what the text says on arkhamdb and what the image of the card says. I'd like someone who has the actual set to confirm which version is correct. If the image is correct though, there is a line break missing after the first part.

Reply:
The image on ArkhamDB is correct.

Reply:
Interesting. These changes were made using the images in-game, which seem to favor the text on ArkhamDB as correct. What could be the case here? Is it possible FFG is showing pre-release versions of some cards on their DB?